### ISSUE TITLE
path parameter rename not working correctly

### ISSUE THREAD

**Issue by masseelch** [Milad-Akarie/auto_route_library#406]
Consider this route:
```dart
AutoRoute(
  path: '/displays',
  guards: [AuthGuard],
  page: EmptyRouterPage,
  children: [
    AutoRoute(
      path: '',
      page: DisplayListPage,
    ),
    AutoRoute(
      path: ':id',
      page: DisplayDetailPage,
    ),
  ],
)

class DisplayDetailPage extends StatefulWidget {
  DisplayDetailPage({@PathParam('id') this.displayId = ''});

  final String displayId;

  @override
  _DisplayDetailPageState createState() => _DisplayDetailPageState();
}
```

The genrated `PageRouteInfo` does not use the renamed segment.

```dart
class DisplayDetailRoute extends _i1.PageRouteInfo<DisplayDetailRouteArgs> {
  DisplayDetailRoute({this.displayId = ''})
      : super(name,
            path: ':id',
            args: DisplayDetailRouteArgs(displayId: displayId),
            params: {'id': id}); // <- should be {'id': displayId}

  final String displayId;

  static const String name = 'DisplayDetailRoute';
}
```

**Comment by Milad-Akarie** [comment]
@masseelch  That was fixed in auto_route 1.0.1 
 can you confirm?

**Comment by masseelch** [comment]
Yes. It is fixed in 1.0.1
